app/assets/javascripts/modules/tabs.js.coffee

```diff
@@ -1 +1,25 @@
+define('tabs', [], ->
+  class Tabs
+    constructor: (@$container) ->
+      @addEventListners()
 
+    addEventListners: ->
+      @$container.find('.js-tab-title').on('click', (e) =>
+        $clicked = e.target
+
+        @updateActiveTabTitle($clicked);
+        @updateActiveTabContent($clicked);
+      )
+
+    updateActiveTabTitle: (tabTitle) ->
+      @$container.find($('.js-tab-title')).removeClass('tab__title--active')
```

Comment:
I think we can avoid instantiating the jquery object `$('.js-tab-title')`, and just use `find('.js-tab-title')`. Should yield the same result!